src/utils/nodes.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
```

Comment:
can we move this to the fields folder and call it textNodes.jsx

Reply:
Done.